submissions/smillims/tiny-js-world/index.js

```diff
@@ -1,44 +1,41 @@
 /* 
-   Code repository: _put repo URL here_
-   Web app: _put project's github pages URL here_
+	Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
 */
 
-
 const inhabits = [
-	{
+	cat = {
 		species: 'cat',
 		name: 'Murka',
 		surname: 'My hosts didn\'t give me a surname',
 		gender: 'female',
-		legs: 'Have 4 legs',
-		hands: 'Cats haven\'t hands :(',
+		legs: '4',
 		saying: 'Hey, I\'m cat',
 	},
-	{
+	man = {
 		species: 'human',
 		name: 'Jorj',
 		surname: 'Kluni',
 		gender: 'male',
-		legs: 'Have 2 legs',
-		hands: 'Have 2 hands',
+		legs: '2',
+		hands: '2',
 		saying:'Did you see Ocean\'s Eleven?',
 	},
-	{
+	man = {
 		species: 'superhuman',
 		name: 'Peter',
 		surname: 'Parker',
 		gender: 'male',
-		legs: 'Have 2 legs',
-		hands: 'Have 2 hands, sometimes more :)',
+		legs: '2',
+		hands: '2',
 		saying: 'Your friendly neighborhood Spider-Man!',
 	},
-	{
+	pig = {
```

Comment:
`a dog, a cat, a woman, and a man.` 

Why don't you follow technical requirements?

Reply:
Wanted to did something unique.
Am I should correct it?

Reply:
@smillims maybe first do minimal requirements ?)